eslint-no-new-dates-plugin.mjs

```diff
@@ -0,0 +1,7 @@
+import noNewDatesRule from './eslint-no-new-dates-rule.mjs';
+
+const noNewDatesPlugin = {
+  rules: { 'no-new-dates': noNewDatesRule },
+};
+
+export default noNewDatesPlugin;
```

Comment:
What do you think about the following?

1. A directory for any custom eslint rules to avoid further cluttering the root directory
2. Calling the plugin something more general like `eslint-custom-rules-plugin.mjs`
3. Adding all custom rules (currently, 'no-new-dates') to `eslint-custom-rules-plugin.mjs`